webapp/src/ts/services/contact-stats.service.ts

```diff
@@ -0,0 +1,89 @@
+import { Injectable } from '@angular/core';
+import * as moment from 'moment';
+import { isObject as _isObject, uniq as _uniq } from 'lodash-es';
+import * as CalendarInterval from '@medic/calendar-interval';
+
+import { DbService } from '@mm-services/db.service';
+import { SessionService } from '@mm-services/session.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ContactStatsService {
```

Comment:
I think we should rename this to something that links it to the UHC feature set, since it only contains methods around UHC home visits.

Reply:
I was thinking of leaving this service as a place for any contact stats, if someone wanted to extend the feature. 
But probably it's too broad and it should be limited

`UHCStatsService` and the contact summary property `uhcStats`, so people will use it like this:
```
uhcStats.homeVisits.count
uhcStats.homeVisits.countGoal
```
is that okay?

Reply:
uhcStats is great!

Reply:
Done!